cluster/images/hyperkube/Dockerfile

```diff
@@ -78,3 +75,7 @@ RUN ln -s /hyperkube /apiserver \
  && ln -s /hyperkube /kubelet \
  && ln -s /hyperkube /proxy \
  && ln -s /hyperkube /scheduler
+
+# Copy the hyperkube binary
+COPY hyperkube /hyperkube
```

Comment:
I guess it was already after the apt commands, but still this exploits the cache slightly better.